mce-label-injector/deployments/deployment.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: v1
```

Comment:
I understand you want to make this info configurable but do we really need to let end user able to change this info ? 
better to not give this option 
OR 
remove the configMap and use the default values as it is defined [here](https://github.com/stolostron/cluster-api-installer/pull/37/files#diff-65888fb6ad080d04f36f5d7a7949f10ba23eb1fba42cea42fa29918655f34175R30)